src/convert.jl

```diff
@@ -97,30 +99,25 @@ function trixi2vtk(filename::AbstractString...;
 
     # Read mesh
     verbose && println("| Reading mesh file...")
-    @timeit "read mesh" (center_level_0, length_level_0,
-                         leaf_cells, coordinates, levels) = read_meshfile(meshfile)
+    # TODO 100_000 could be too small in some cases
+    @timeit "read mesh" mesh = Trixi.load_mesh_serial(meshfile; RealT=Float64, n_cells_max=100_000)
```

Comment:
```suggestion
    @timeit "read mesh" mesh = Trixi.load_mesh_serial(meshfile; RealT=Float64)
```
Can be omitted with Trixi v0.3.31